packages/@aws-cdk/aws-ec2/test/port.test.ts

```diff
@@ -0,0 +1,16 @@
+import { Port, Protocol } from '../lib';
+
+describe('port', () => {
+  describe('Port class', () => {
+    describe('.allIcmpV6', () => {
+      it('returns expected instance of Port', () => {
+        expect(Port.allIcmpV6()).toEqual(new Port({
```

Comment:
Let's check this against the template generated, not against the data structure itself.